lib/extensions/method.rb

```diff
@@ -108,7 +114,7 @@ def validate_args!
           Method::WithArgs arguments don't match the method signature of the source method:\n\n
           #{source}\n\n
           (NOTE: unbracketed hash arguments aren't supported)
-        ).squish
+        ).split.join(' ')
```

Comment:
Initial code was functional tested within rails application, but this gem doesn't require rails. This just squishes without the sugar.